README.md

```diff
@@ -1,2 +1,76 @@
 # ra-auth-google
-Auth provider for react-admin using the Google Identity Services
+
```

Comment:
Could it be a good idea to define which are the prerequisites (Node, yarn, ...)?